docs/rules/require-await.md

```diff
@@ -18,6 +18,7 @@ async function fetchData(processDataItem) {
 
 Asynchronous functions that don't use `await` might not need to be asynchronous functions and could be the unintentional result of refactoring.
 
+> This rule doesnt report for non `await` async generator functions
```

Comment:
The double negative is a bit confusing here. Can we change this to
```suggestion
Note: this rule ignores async generator functions. This is because ...
```
and explain why they're ignored?